src/Server/datadog.ts

```diff
@@ -1,7 +1,6 @@
-import ddTracer from "dd-trace"
+import ddTracer, { Span } from "dd-trace"
```

Comment:
This fixed a type change that was introduced in v2.31.0. See [this](https://artsy.slack.com/archives/CA8SANW3W/p1682707134740739?thread_ts=1682540612.539359&cid=CA8SANW3W) slack comment for more info.